batch-swagger.yaml

```diff
@@ -275,7 +275,7 @@ paths:
     x-summary: Spaces
     get:
       operationId: listSpacesForAccount
-      summary: List spaces for an account
+      summary: List collections for an account
       description: Returns all `Space`s for an `Account`
```

Comment:
This change is _probably_ ok, but unrelated to the rest of this PR. I say "probably" because we're still using Space for the route and the name of the object we're returning. Does changing the summary to read "collection" actually help our users?

Maybe more of a question for the Admin & Growth team.

Reply:
When working on form fields I noticed that this change was made manually in an auto-generated file in the developer docs repo. I made a separate PR to update the swagger blocks in webapp that was reviewed by Admin & Growth so their manual changes wouldn't be overwritten (reverted). I guess I could have pulled this out into a separate PR for this repo, but it seemed trivial.